.gitlab/docker_common/tag_job_templates.yml

```diff
@@ -93,6 +93,7 @@
   variables:
     <<: *google_container_registry_variables
   before_script:
+    - '$_instance_id = (iwr  -UseBasicParsing http://169.254.169.254/latest/meta-data/instance-id).content ; Write-Host "Running on instance $($_instance_id)"'
```

Comment:
```suggestion
    - $_instance_id = (iwr  -UseBasicParsing http://169.254.169.254/latest/meta-data/instance-id).content
    - Write-Host "Running on instance $($_instance_id)"
```

Reply:
If you want to do that, we should do it in a separate PR. This line appears a lot of times across all our gitlab files 😄